app/components/FormLine.js

```diff
@@ -9,15 +9,14 @@ const FormLine = ({ label, value, valueUpdateCB }) => {
   //console.log('VALUE UPDATE CB IN FORM LINE ' + typeof valueUpdateCB);
   const removeNonNumericValues = (label, value) => {
     if (!value) {
-      return;
-    }
-
-    const regexNotNumberOrDot = /[^0-9.]/g;
-    const regexLeadingNumber = /^0+/;
-
-    value.replace(regexNotNumberOrDot, "");
-    value.replace(regexLeadingNumber, "");
+      value = 0;
```

Comment:
Setting default value of empty input field to 0 allows users to backspace to empty input field.
Before, user could add an input, and would not be able to backspace to empty field.